src/useWith.js

```diff
@@ -44,20 +44,18 @@ var curry = require('./curry');
  *      // If there are extra _expected_ arguments that don't need to be transformed, although
  *      // you can ignore them, it might be best to pass in the identity function so that the new
  *      // function correctly reports arity.
- *      var addDoubleAndSquareWithExtraParams = R.useWith(addAll, double, square, R.identity);
+ *      var addDoubleAndSquareWithExtraParams = R.useWith(addAll, [double, square, R.identity]);
  *      // addDoubleAndSquareWithExtraParams.length //=> 3
  *      //≅ addAll(double(10), square(5), R.identity(100));
  *      addDoubleAndSquare(10, 5, 100); //=> 145
  */
-module.exports = curry(function useWith(fn /*, transformers */) {
-  var transformers = _slice(arguments, 1);
-  var tlen = transformers.length;
-  return curry(_arity(tlen, function() {
+module.exports = curry(function useWith(fn, transformers) {
```

Comment:
We have an optimized [`_curry2`][1] function for currying binary functions. Let's use it here in place of `curry`.


[1]: https://github.com/ramda/ramda/blob/v0.17.1/src/internal/_curry2.js